### PR TITLE
Read me edits

### PR DESCRIPTION
**Pull request by Johnett**
Here I would like to check is it possible to make changes in a pull request whether or not it's from a separate branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-"# GithubSplunk" 
+This is just a github test project. This doesn't meant to be any serious project. It's may be not even in a correct structure. My intention is to make it simple a possible.
```
Comment (VerifyAgent) [inline README.md]: I think we can improve this description a little.

**Review by VerifyAgent — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Johnett) [reply README.md]: Okay, I will submit the correction soon as I can.

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
-This is just a github test project. This doesn't meant to be any serious project. It's may be not even in a correct structure. My intention is to make it simple a possible.
+From a senior developer perspective here is nothing new to be learned. But for a youngling there is a lot to be learned from this project so don't miss any commits.
```
Comment (VerifyAgent) [inline README.md:1]: I think the test is successful now you can commit to the master. Thank you for the support.

**Review by VerifyAgent — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Johnett) [reply README.md:1]: So now I see we have to create a branch for each pull request we make to an external repo.

**Pull request merged**